NOTICE.txt

```diff
@@ -1006,6 +1006,50 @@ Files covered:
 spyder/plugins/variableexplorer/widgets/dataframeeditor.py
 
 
+-------------------------------------------------------------------------------
+
+
+
+Classes from objbrowser 1.2.1
+-----------------------------
+
+
+Copyright (c) 2016 Pepijn Kenter
+
+
+Author: Pepijn Kenter | <author-email> | <author-website>
```

Comment:
Please try to add `author-email` here (you can remove `author-website` if you can't find one quickly).